### PR TITLE
fix(ui):  Rename 'Issue Ownership' to 'Issue Owners.' Append a 'new' badge.

### PR DESCRIPTION
**Pull request by ckj**
screen shot 2018-04-30 at 2 30 06 pm

Closes APP-77.

### CODE REVIEW

**Comment by mattrobenolt** [comment]
:fire:

File: src/sentry/static/sentry/app/views/settings/components/settingsNavItem.jsx [hunk 1]
```diff
@@ -39,6 +40,15 @@ const StyledNavItem = styled(Link)`
   }
 `;
 
+const NewBadge = styled(Badge)`
```
Comment (billyvg) [inline src/sentry/static/sentry/app/views/settings/components/settingsNavItem.jsx]: Do we need any of the CSS from `Badge`, design-wise, they seem pretty different

File: src/sentry/static/sentry/app/views/settings/project/navigationConfiguration.jsx [hunk 2]
```diff
@@ -52,8 +52,11 @@ export default function getConfiguration({project}) {
         },
         {
           path: `${pathPrefix}/ownership/`,
-          title: t('Issue Ownership'),
-          description: t('Manage code ownership rules for a project'),
+          title: t('Issue Owners'),
+          description: t('Manage issue ownership rules for a project'),
+          badge: () => {
```
Comment (billyvg) [inline src/sentry/static/sentry/app/views/settings/project/navigationConfiguration.jsx]: nit: `badge: () => 'new',`

**Review by billyvg — APPROVED** [review]

File: src/sentry/static/sentry/app/views/settings/components/settingsNavItem.jsx (hunk 1 above)
Comment (ckj) [reply src/sentry/static/sentry/app/views/settings/components/settingsNavItem.jsx]: I started down that path, but i think there were enough things... positioning, text size, props -> children stuff, that badge provides to make this route slightly less annoying.

**Pull request merged**